configure

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-$R_HOME/bin/Rscript inst/cpp_generation/generate.R
+##$R_HOME/bin/Rscript inst/cpp_generation/generate.R
```

Comment:
Had to turn this off for now as the structure of some of the generated code has changed. It is going to change again (fairly noisily) I expect as we add the Python interface. So I am tempted to keep this off for now until we settle on the interface again